subjects/road_intersection/audit/README.md

```diff
@@ -1,86 +1,98 @@
 
-##### Try to generate two vehicles at the same time, one using the `"Up"` key and the other using the `"Left"` key (to this at least 3 times).
+##### Try to generate two vehicles at the same time, one using the `"Up"` key and the other using the `"Left"` key (do this at least 3 times).
 
-###### Did all vehicles passed the intersection without any collision?
+###### Did all the vehicles passed the intersection without any collision?
 
-##### Try to generate two vehicles at the same time, one using the `"Up"` key and the other using the `"Right"` key (to this at least 3 times).
+##### Try to generate two vehicles at the same time, one using the `"Up"` key and the other using the `"Right"` key (do this at least 3 times).
 
-###### Did all vehicles passed the intersection without any collision?
+###### Did all the vehicles passed the intersection without any collision?
 
-##### Try to generate two vehicles at the same time, one using the `"Down"` key and the other using the `"Left"` key (to this at least 3 times).
+##### Try to generate two vehicles at the same time, one using the `"Down"` key and the other using the `"Left"` key (do this at least 3 times).
 
-###### Did all vehicles passed the intersection without any collision?
+###### Did all the vehicles passed the intersection without any collision?
 
-##### Try to generate two vehicles at the same time, one using the `"Down"` key and the other using the `"Right"` key (to this at least 3 times).
+##### Try to generate two vehicles at the same time, one using the `"Down"` key and the other using the `"Right"` key (do this at least 3 times).
 
-###### Did all vehicles passed the intersection without any collision?
+###### Did all the vehicles passed the intersection without any collision?
 
-##### Try to generate five vehicles using the `"Up"` key, at the same time generate two vehicles using the key `"Right"`.
+##### Try to generate five vehicles using the `"Up"` key, at the same time generate two vehicles using the `"Right"` key.
 
-###### Did all vehicles passed the intersection without any collision?
+###### Did all the vehicles passed the intersection without any collision?
 
-##### Try to generate one vehicle for all the lanes (to this at least 3 times).
+##### Try to generate one vehicle for all the lanes (do this at least 3 times).
 
-###### Did all vehicles passed the intersection without any collision?
+###### Did all the vehicles passed the intersection without any collision?
 
 ##### Try to generate vehicles randomly using the `"R"` key. Then wait for at least 1 min.
 
-###### Did all vehicles passed the intersection without any collision?
+###### Did all the vehicles passed the intersection without any collision?
 
-###### Was there a low traffic congestion while running the application? (ex: high traffic congestion can be 8 or more vehicles in the same lane without proceeding)?
+##### Try to generate multiple vehicles with lanes at your choice.
```

Comment:
Same as the comment for line 63 (although I may have misunderstood the concept of lanes).

Reply:
My comment is irrelevant. We're using single carriageways.